rfc/src/template.md

```diff
@@ -10,47 +10,66 @@
 
 ## Summary
 
-Short description of the feature, i.e.: the elevator pitch. What is this feature about?
+Short (1-2 sentences) description of the feature. What is this feature about?
 
 ## User Impact
 
 Why are we doing this? How will this benefit the final user?
 
  - If this is an API change, how will that impact current users?
- - For deprecation or breaking changes, how will the transition look like?
  - If this RFC is related to change in the architecture without major user impact, think about the long term
-impact for user. I.e.: what future work will this enable.
+impact for user. I.e.: what future work will this enable. Maybe reach out to other developers to ensure you  need an RFC in the first place.
 
 ## User Experience
 
-What is the scope of this RFC? Which use cases do you have in mind? Explain how users will interact with it. Also
-please include:
+This should be a description on how users will interact with the feature.
+Users should be able to read this section and understand how to use the feature.
+**Do not include implementation details in this section, neither discuss the rationale behind the chosen UX.**
 
-- How would you teach this feature to users? What changes will be required to the user documentation?
-- If the RFC is related to architectural changes and there are no visible changes to UX, please state so.
+Please include:
+  - High level user flow description
+  - Any new major functions or attributes that will be added to Kani library.
+  - New command line options or subcommands (no need to mention the unstable flag).
+  - What are possible user errors and how are they presented. E.g: Compilation vs Verification error.
+  - Substantial changes to existing functionality or Kani output.
 
-## Detailed Design
+If the RFC is related to architectural changes and there are no visible changes to UX, please state so.
+No further explanation is needed.
 
-This is the technical portion of the RFC. Please provide high level details of the implementation you have in mind:
+## Software Design
 
-- What are the main components that will be modified? (E.g.: changes to `kani-compiler`, `kani-driver`, metadata,
-  installation...)
-- How will they be modified? Any changes to how these components communicate?
-- Will this require any new dependency?
+**We recommend you to leave this empty for the first version of your RFC**.
```

Comment:
I would not lead with this. I would lead with the one-sentence-summary, e.g. "This is the technical ...., then say that this should only contain any major changes to the architecture and then add the "We recommend you to leave this empty ..." sentence

Reply:
Sure. That's what I had initially. :)